app/mailers/custom_devise_mailer.rb

```diff
@@ -7,4 +8,8 @@ def confirmation_instructions(record, token, options = {})
     @confirmation_period = user_decorator.confirmation_period
     super
   end
+
+  def attach_images
+    attachments.inline['logo.png'] = File.read('app/assets/images/logo.png')
+  end
```

Comment:
What is the reasoning behind attaching the image as opposed to reading it from the asset pipeline like we were doing before?